package.json

```diff
@@ -56,6 +56,7 @@
     "crypto-browserify": "^3.12.1",
     "date-fns": "^4.1.0",
     "dom-to-image-more": "^3.5.0",
+    "dompurify": "^3.2.6",
```

Comment:
I don't think we need a new npm package for the FAQ page. This seems like a sign you are overcomplicating the implementation.